app/components/article-content.js

```diff
@@ -388,6 +391,51 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Opens a parent section of passed element if it's closed
+		 *
+		 * @returns {void}
+		 */
+		openSection(element) {
+			if (element) {
+				const section = element.closest('section[id*="section"]');
+				if (section) {
+					const header = section.previousElementSibling;
+					if (header) {
+						header.classList.add('open-section');
```

Comment:
What about scrolling the ref into view after expanding the section? Does that work now based on url hash or should we think of adding some JS to scroll the given ref into view?